Tools/setup/ubuntu.sh

```diff
@@ -126,9 +126,11 @@ if [[ $INSTALL_NUTTX == "true" ]]; then
 		genromfs \
 		gettext \
 		gperf \
+		fuse \
 		kconfig-frontends \
 		libelf-dev \
 		libexpat-dev \
+		libfuse2 \
 		libgmp-dev \
```

Comment:
@mrpollo instead of `libfuse2` and `fuse`, could the newer `fuse3`? Or is there a particular reason for using the other two?